README.md

```diff
@@ -161,6 +161,33 @@ audience, it's useful to build on a system with older library versions. Python
 has a useful pointers in their [packaging
 guides](https://packaging.python.org/guides/packaging-binary-extensions/#building-binary-extensions).
 
+### Static Compiling OpenSSL into Erlang Distribution
+
+Sometimes wierd SSL state bugs arise with a release when openssl is not statically compiled in.
+Your affected by these ssl issues if you see things like a ssl connection being established but
```

Comment:
```suggestion
You're affected by these ssl issues if you see things like an SSL connection being established but
```

Reply:
The "a ssl" connection is the rule not to use an if the next work starts with a vowel?

```
Use “a” before words that start with a consonant sound and “an” before words that start with a vowel sound.
```

Reply:
Ya, so this is prob more specific to me. There is lots of debate about focusing on the sound vs the starting letter and in almost every case I'm involved with, capitalized acronyms (SSL, etc) get an "an", so that has become my default and habit.

But really, I'd be good either way and more just prefer the capitalized acronym

Reply:
sounds good with me, pushed